javascript/utils.js

```diff
@@ -207,6 +207,9 @@ function __resource_type_num_to_str(num) {
         case 83: return RESOURCE_EMANATION;
         case 84: return RESOURCE_ESSENCE;
         case 1001: return SUBSCRIPTION_TOKEN;
+        case 1002: return CPU_UNLOCK;
```

Comment:
Is it just me or are all those constants gone from the screeps doc? We could just now encode this in an enum instead.

Not in this PR, but still.

Reply:
Hrmm not sure if I'm following - should still be in their docs? https://github.com/screeps/common/blob/master/lib/constants.js#L379

Reply:
You're right, I checked their history and I imagined things. I thought the integers were related to similar constants in the js implementation. It seems we only use them to make the translation easier.